pkg/minikube/bootstrapper/kubeadm/kubeadm.go

```diff
@@ -332,8 +334,29 @@ func (k *Bootstrapper) WaitForNode(cfg config.ClusterConfig, n config.Node, time
 		glog.Infof("%s is not a control plane, nothing to wait for", n.Name)
 		return nil
 	}
+
+	// starting from minikube v1.10 checking for node pressure is a reuqired check
```

Comment:
Remove this comment: it doesn't describe a "why" at all and is generally not helpful. Also, spelling.